tests/behat/features/Request.feature

```diff
@@ -0,0 +1,91 @@
+@request
+Feature: Request information from an agency
+  In order to obtain information
+  As an end user
+  I should be able to submit a request
+
+  @api
+  Scenario: Send request via email
+    Given I am logged in as a user with the 'Administrator' role
+
+    ### Create form
+
+    And I am at 'admin/structure/webform/add'
+    And for 'Machine-readable name' I enter 'test_webform'
+    And for 'Title' I enter 'Test webform'
+    And I press the 'Save' button
+    And I am at 'form/test-webform'
+    And for "First Name" I enter "Ringo"
+    And for 'Last name' I enter 'Star'
+    And for 'Email' I enter 'test@example.com'
+    And for "Describe the information you're requesting" I enter 'Test description'
+    And I select "No" from "Request Fee Waiver"
+    And I select "No" from "Request Expedited Processing"
+    And I press the 'Submit' button
+
+    ### Error I (No associated agency component)
+
+    When I am at 'admin/reports/dblog'
+    Then I should see the text 'Unassociated form: The form, Test webform'
+
+    ### Create agency component for testing.
+
+    And I am at 'node/add/agency_component'
+    And for 'Agency Component Name' I enter 'Test agency component'
+
+    ### Associate request form.
+
+    And I select "Test webform" from "Request Submission Form"
+
+    ### Error II (No associated email)
```

Comment:
This isn't a big enough issue to fail the ticket, but should this failure case test not be a separate test?